addons/sales_team/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,8 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
```

Comment:
```suggestion
@openupgrade.migrate()
```
by default is `use_env=True`